### DISCUSSION TITLE
Not getting messages for FB inbox even after configuring through given facebook guide on site

### DISCUSSION THREAD

**Discussion by muzammal-murtaza** [chatwoot/chatwoot#2072] [Q&A]
**Describe the bug**

I am not getting inbox messages for FB inbox.

**Comment by muzammal-murtaza** [comment]
should i need to add some environment vars?

**Comment by pranavrajs** [comment]
Please share the steps you have followed. Have you sent the FB app for verification?

**Comment by muzammal-murtaza** [comment]
Yes i have verified the app too and follow all these steps,

In the app settings add your Chatwoot installation url as your app domain.
In the products section in your app settings page, Add Messenger
Go to the Messenger settings and configure the call Back URL with {your_chatwoot_url}/bot
Configure a verify token, you will need this value for configuring the chatwoot environment variables
You might have to add a Facebook page to your Access Tokens section in your Messenger settings page if your app is still in development.
You will also have to add your Facebook page to webhooks sections in your messenger settings with all the webhook events checked.

**Reply by urumat** [reply]
Hello, could you please tell me how you verified the app? What link did you use for privacy policy, and terms of service ?

**Comment by pranavrajs** [comment]
@muzammal-murtaza Can you please share server logs after you sent a message to FB Page? There is too little information here to debug the problem.

**Comment by amblerkr** [comment]
I tried it, but I think there's the same problem. It would be nice to have a guide video. I might have done something wrong.

**Comment by pranavrajs** [comment]
Moving this to the discussions as I believe it is not a bug. 

@amblerkr Yes, it makes sense to add a guide video.

**Reply by helpobras** [reply]
Hello..happened to me now in a self-hosting.
authenticated... my page to choose... appeared.
but don't receive the messages

**Reply by sojan-official** [reply]
please check https://github.com/chatwoot/chatwoot/issues/2818
